### DISCUSSION TITLE
Cant get mods.txt working

### DISCUSSION THREAD

**Discussion by marqxb** [itzg/docker-minecraft-server#1590] [Q&A]
Cannot figure out why i get

[init] Resolved version given LATEST into 1.19 and major version 1.19
[init] Resolving type given FORGE
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x. 1 1000 1000 324 Jul  8 11:37 /data'
[init] Checking Forge version information.
[init] ERROR: given MODS_FILE file does not exist
[mc-image-helper] 12:46:55.016 ERROR : 'get' command failed: Missing property in path $.promos['1.19-recommended']

**Comment by itzg** [comment]
Those logs are in a really weird order. Is that exactly how they were reported? Can you provide your container config?

**Reply by marqxb** [reply]
version: '3.8'

services:
  mc:
    image: itzg/minecraft-server
    environment:
      EULA: "true"
      TYPE: FORGE
      MODS_FILE: /extras/mods.txt
      REMOVE_OLD_MODS: "true"
    ports:
      - 25565:25565
    volumes:
      - data:/data
      - ./mods.txt:/extras/mods.txt:ro

volumes:
  data:

**Reply by marqxb** [reply]
oh and I am using Portainer if that makes any difference

**Reply by marqxb** [reply]
Oh damn check when you copy/paste stuff... 

[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x. 1 1000 1000 324 Jul  8 11:37 /data'
[init] Resolved version given LATEST into 1.19 and major version 1.19
[init] Resolving type given FORGE
[init] Checking Forge version information.
[mc-image-helper] 12:46:55.016 ERROR : 'get' command failed: Missing property in path $.promos['1.19-recommended']
[init] ERROR: given MODS_FILE file does not exist

**Comment by itzg** [comment]
The mc-image-helper error threw me off, but trying myself I realized that's not a real error.

As for the MODS_FILE error, can you double check that you do have a `mods.txt` file in the same directory as your `docker-compose.yml` file?
